src/nuget/xunit.console.netcore.nuspec

```diff
@@ -33,11 +33,21 @@
       <dependency id="System.Threading" version="4.0.10" />
       <dependency id="System.Threading.Tasks" version="4.0.10" />
       <dependency id="System.Xml.XDocument" version="4.0.10" />
+      <dependency id="xunit.runner.utility" version="2.1.0-rc1-build3168" />
     </dependencies>
   </metadata>
   <files>
     <!-- xunit runner for .NET Core -->
     <file src="xunit.console.netcore\xunit.console.netcore.exe" target="lib\aspnetcore50" />
+    <file src="xunit.console.netcore\_._" target="lib\dnxcore50" />
+    <file src="xunit.console.netcore\_._" target="lib\dotnet" />
+    
+    <!--
+      NuGet ignores .exe files in lib folders, so use .props files to copy the exe
+      to the output folder.
+    -->
+    <file src="xunit.console.netcore\xunit.console.netcore.exe" target="build\dotnet" />
+    <file src="xunit.console.netcore\xunit.console.netcore.props" target="build\dotnet" />
   </files>
```

Comment:
> native\xunit.console.netcore.exe

@ericstj I tried this, but it didn't work.  It goes in project.lock.json as expected, but the ResolveNuGetPackageAssets task doesn't deploy anything from a native folder that's not specific to a runtime.  The logic is in [GetCopyLocalItems](https://github.com/dotnet/roslyn-internal/blob/275671ee97283bbff71b59e173ed9319ac415b83/Closed/MSBuildNuGet/Tasks/ResolveNuGetPackageAssets.cs#L234).

I'm not sure a fix for this would go entirely in the task, either.  You'd need to define what happens when you have a native folder that's specific to a runtime a runtime-agnostic one.  Should you take all files from both (preferring the runtime-specific ones when there is the same file in both places), or should the existence of a runtime-specific native folder mean the runtime-agnostic folder isn't used at all?  That type of decision seems like maybe it should be part of NuGet instead of the ResolveNuGetPackageAssets task.